pages/pinner-award.md

```diff
@@ -0,0 +1,51 @@
+title: The John Pinner Award
+---
+<h2>What?</h2>
+
+The annual John Pinner award celebrates voluntary service to the Python
+community in the UK. It's for those who freely contribute time and effort for
+the good of the community but who generally don't get the sort of recognition
+others may enjoy.
+
+This is an annual award with up to three awards being given at once. The awards
+will be announced at the start of each annual PyCon UK conference, so
+recipients at the conference may enjoy a weekend of recognition from the wider
+community in attendence.
+
+It is only possible to win the award once.
+
+Recipients of the award will receive a free ticket to next year's PyCon UK and
+a token in recognition of their efforts (most likely a medal of some sort).
+
+<h2>Why?</h2>
+
+John Pinner founded and organised PyCon UK until he passed away in 2015. He was
+the personification of a Pythonista who gave time, code and effort back to the
+community. He was a role model, mentor and teacher who is sorely missed by
+many.
+
+With the blessing of the Pinner family, the UKPA is initiating the award to
+honour and remember John's extraordinary contribution to our community.
+
+<h2>How?</h2>
+
+The award is administered by at least two returning officers. The returning
+officers may not receive the award and any nominations for them will be
+discarded. For this year the returning officers are Nicholas Tollervey and
+Peter Inglesby.
+
+Nominations from the wider Python UK community are made simply by filling in
+the form embedded below. The returning officers will confirm
+each nomination meets the requirements outlined above (specifically, that the
+nomination is for Python related community service in the UK and the nominator
+is a real person).
+
+To win an award a person needs to have at least three separate nominations. The
+recipients of the award will be the three most nominated people in any
+particular annual nomination period of the award.
```

Comment:
In the case of ties, may I suggest that we simply allow for more recipients of the award since we want to reflect the generous and inclusive nature of the Python community. The notion of "three" awards per-year is simply a rule-of-thumb. After all, if there's a tie at the Olympics, both parties get a medal.

Does this make sense?